subsys/debug/coredump/Kconfig

```
@@ -28,6 +28,13 @@ config DEBUG_COREDUMP_BACKEND_FLASH_PARTITION
 	  Core dump is saved to a flash partition with DTS alias
 	  "coredump-partition".
 
+config DEBUG_COREDUMP_BACKEND_RETAINED_MEM
+	bool "Use retained mem subsys for coredump"
```

```suggestion
	bool "Use retained memory for coredump"
```
Retained memory is a driver, one important thing to note is that retained memory will take up a whole area, e.g. if you use a 10KB RAM area for it, it is the whole 10KB RAM area, you can't only use e.g. 2KB and then use 8KB for something else. The data retention subsystem builds on top of retained memory and does add this support, it also adds an optional checksum and optional magic header to the data area - it is currently in review at https://github.com/zephyrproject-rtos/zephyr/pull/55125

---

any ideas on when #55125 should get merged ?

---

Unfortunately not, it seems to be suffering from a lack of people that want to review it, feel free to review it though :)

---

Didn't realise I could review it, I'm not a maintainer